backend/src/service/Jimmi.service.ts

```diff
@@ -1,14 +1,53 @@
 import Jimmi from './Jimmi.class.ts';
+import config from '../configuration/environment.ts';
+import puppeteer, { Browser } from 'https://deno.land/x/puppeteer@9.0.2/mod.ts';
 
 const instances: Record<string, Jimmi> = {};
+const args = [
+  '--use-fake-ui-for-media-stream', // disable asking for webcam & video
+  '--use-fake-device-for-media-stream', // use fake microphone
+  '--disable-web-security', // enable playback of cross origin media/resources
+  '--disable-features=IsolateOrigins,site-per-process', // allow to access cross-origin iframe
+  // performance related
+  '--disable-accelerated-2d-canvas',
+  '--single-process',
+  '--disable-gpu',
+];
+
+let browser: Browser;
+try {
+  if (config.browser.wsEndpoint !== undefined) {
+    console.info('Using chrome on remote endpoint!');
+    browser = await puppeteer.connect({
+      browserWSEndpoint: config.browser.wsEndpoint,
+    });
+  } else {
+    if (config.browser.noSandbox) {
+      args.push('--no-sandbox');
+      console.warn('Warning: Browser started with --no-sandbox flag!');
+    }
+
+    console.info('Using chrome on local instance');
+    browser = await puppeteer.launch({
+      executablePath: '/usr/bin/chromium',
```

Comment:
Should we make the executable path configurable?

Reply:
Solved in [f225042](https://github.com/Music-Bot-for-Jitsi/Jimmi/pull/117/commits/f22504250a30231d784837d1469ed41f2d2923fc)